tests/test_routes.py

```diff
@@ -0,0 +1,106 @@
+from tests.helpers import count_fund_applications
```

Comment:
Absolutely beautiful tests - nice to have the code duplication cleared up by the wonderful test helpers 😻